viewer/packages/data-providers/src/image-360-data-providers/Cdf360ImageEventProvider.ts

```diff
@@ -349,19 +350,19 @@ export class Cdf360ImageEventProvider implements Image360Provider<Metadata> {
         annotationType: 'images.AssetLink'
       });
 
-      const assetIds = annotationArray
+      const assetAnnotations = annotationArray
         .filter(annotation => annotationFilter(annotation))
         .map(annotation => {
           assert(isAssetLinkAnnotationData(annotation.data), 'Received annotation that was not an assetLink');
-          return annotation.data.assetRef;
+          return annotation as ImageAssetLinkAnnotationInfo;
```

Comment:
Would prefer type guard. Almost tempted to not allow `as` in a lint rule. You might be able to merge it into the annotationsFilter if you are a little clever with types, such that you don't need an additional map/filter.